components/Dialogs/examples/DialogsCustomShadowExampleViewController.swift

```diff
@@ -97,7 +102,7 @@ class DialogsCustomShadowExampleViewController: UIViewController {
     // We set the corner radius to adjust the shadow that is implemented via the trackingView in the
     // presentation controller.
     if let presentationController = presentedController.mdc_dialogPresentationController {
-      presentationController.dialogCornerRadius = presentedController.view.layer.cornerRadius
+      presentationController.applyTheme(withScheme: containerScheme)
```

Comment:
When applying the theme, how do you ensure the corner radius of the presentation controller is equal to the presented controller's one?   
Also, the leading comment doesn't match what's being done anymore.

Reply:
I _re-added_ the line that was removed.